suites/reef/cephadm/tier4-service-operations.yaml

```diff
@@ -171,3 +171,9 @@ tests:
       desc: Validate the key rotation feature
       polarion-id: CEPH-83593829
       module: test_validate_key_rotate.py
+
+  - test:
```

Comment:
Hi @pranavprakash20 

Only applicable to reef?

Can we add same test to squid as well?

Reply:
@MohitBis Noted. Added